packages/core/utils/src/ssr/index.ts

```diff
@@ -2,9 +2,9 @@ import { getCurrentInstance, onServerPrefetch } from '@vue/composition-api';
 
 type ResourceFunction<T> = (params: any) => Promise<T>
 
-const getRootState = (vm) => {
+const getRootState = (vm: any) => {
   if (vm.$isServer) {
-    return vm.$ssrContext.nuxt.vsfState || {};
```

Comment:
porque?

Reply:
line 19 covers this case. So it was not possible to have this alternative executed :)